Source/Scene/ImageryProvider.js

```diff
@@ -338,6 +338,10 @@ ImageryProvider.prototype.pickFeatures = function (
 
 const ktx2Regex = /\.ktx2$/i;
 
+// TODO: This can definitely return a (Promise for) ImageBitmap, and also
```

Comment:
I guess this is really a continuation of my question above.  Basically the documented return type is at least incomplete, and might actually be wrong as well.  If all ImageryProviders inherited from ImageryProvider, this would be caught automatically.

Reply:
I'm trying to address this by exporting a typedef to represent all the types an ImageryProvider might return.  So, providers that punt to `ImageryProvider.loadImage` can just resolve as the typedef, but providers that always return one known type can be more specific in their return annotation.